fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/IcebergInsertExecutor.java

```diff
@@ -51,21 +52,27 @@ public void setCollectCommitInfoFunc() {
         coordinator.setIcebergCommitDataFunc(transaction::updateIcebergCommitData);
     }
 
+    @Override
+    protected void beforeExec() {
+        String dbName = ((IcebergExternalTable) table).getDbName();
+        String tbName = table.getName();
+        SimpleTableInfo tableInfo = new SimpleTableInfo(dbName, tbName);
+        IcebergTransaction transaction = (IcebergTransaction) transactionManager.getTransaction(txnId);
+        transaction.pendingCommit(tableInfo);
+    }
+
     @Override
     protected void doBeforeCommit() throws UserException {
+        String dbName = ((IcebergExternalTable) table).getDbName();
+        String tbName = table.getName();
+        SimpleTableInfo tableInfo = new SimpleTableInfo(dbName, tbName);
         IcebergTransaction transaction = (IcebergTransaction) transactionManager.getTransaction(txnId);
-        loadedRows = transaction.getUpdateCnt();
```

Comment:
Why is the update count not showing now?

Reply:
OK I will increase the count stat